manifests/latest/vsphere-7.0u1/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -38,10 +38,10 @@ spec:
             - mountPath: /csi
               name: socket-dir
         - name: csi-resizer
-          image: quay.io/k8scsi/csi-resizer:v1.0.0-rc2
+          image: quay.io/k8scsi/csi-resizer:v1.0.0
           args:
             - "--v=4"
-            - "--csiTimeout=300s"
+            - "--timeout=300s"
             - "--csi-address=$(ADDRESS)"
             - "--leader-election"
```

Comment:
Can we also add new parameters for QPS and burst? https://github.com/kubernetes-csi/external-resizer/blob/master/cmd/csi-resizer/main.go#L62

Reply:
Default values for `kube-api-burst` and `kube-api-qps` will be set. Can we add needed QPS and Brust recommendation in separate PR after validation?

Reply:
yup sounds good!